extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -398,25 +402,11 @@ export class AmpImageViewer extends AMP.BaseElement {
 
   /** @private */
   setupGestures_() {
-    this.gestures_ = Gestures.get(
-        this.element,
-        /* opt_shouldNotPreventDefault */true
-    );
-
-    this.gestures_.onPointerDown(gesture => {
-      if (this.motion_) {
-        this.motion_.halt();
-        gesture.event.preventDefault();
-      }
-    });
+    this.gestures_ = Gestures.get(this.element);
```

Comment:
Note that now `opt_shouldNotPreventDefault` is set to false per library defaults.